unity/src/native/steamaudio_unity_native.cpp

```diff
@@ -183,6 +184,11 @@ void UNITY_AUDIODSP_CALLBACK iplUnityRemoveSource(IPLint32 handle)
     SteamAudioUnity::gSourceManager->removeSource(handle);
 }
 
+void UNITY_AUDIODSP_CALLBACK setGlobalHRTFDisabled(bool disabled)
```

Comment:
For consistency with existing API functions, I would rename this to `iplUnitySetGlobalHRTFDisabled`.

Reply:
Done via bf260e754563f104c5d4eb0efed7acb3e340184f